app/controllers/project.py

```diff
@@ -204,17 +204,51 @@ def get(self):
         page = request.args.get('page', 1, type=int)
         per_page = request.args.get('per_page', 10, type=int)
         keywords = request.args.get('keywords', '')
+        disabled = request.args.get('disabled')
+        project_type = request.args.get('project_type')
+        organisation = request.args.get('organisation')
+
 
         project_schema = ProjectSchema(many=True)
         projects = []
+        pagination_data = {}
+
+        filter_mapping = {
+        'disabled': disabled,
+        'project_type': project_type,
+        'organisation': organisation
+        }
+
+        # count items per project category 
+        project_metadata = {}
+        for category in filter_mapping.keys():
+            distinct_counts = (Project.query.with_entities(getattr(Project, category), func.count(getattr(Project, category))).group_by(getattr(Project, category)).all())
+            project_metadata[category] = {key: value for key, value in distinct_counts}
+
+
+        # Identify which attribute to filter on
+        attribute, attribute_value = next(((k, v) for k, v in filter_mapping.items() if v), (None, None))
```

Comment:
About this . What if at a point we need say disabled research projects at the same point . How can we account for it when only one attribute is given priority at a time . Just thought about it ..